taca/storage/storage.py

```diff
@@ -178,35 +179,56 @@ def cleanup_uppmax(site, days, dry_run=False):
         projects = [ p for p in os.listdir(root_dir) if re.match(filesystem.PROJECT_RE,p) ]
         list_to_delete = get_closed_projects(projects, pcon, days)
+        ## delete and log
+        for item in list_to_delete:
+            if dry_run:
+                logger.info('Will remove {} from {}'.format(item,root_dir))
+                continue
+            try:
+                shutil.rmtree(os.path.join(root_dir,item))
+                logger.info('Removed project {} from {}'.format(item,root_dir))
+                with open(log_file,'a') as to_log:
+                    to_log.write("{}\t{}\n".format(item,datetime.strftime(datetime.now(),'%Y-%m-%d %H:%M')))
+            except OSError:
+                logger.warn("Could not remove path {} from {}"
+                        .format(item,root_dir))
+                continue
     else:
         ##work flow for cleaning archive ##
-        list_to_delete = []
-        archived_in_swestore = filesystem.list_runs_in_swestore(path=CONFIG.get('cleanup').get('swestore').get('root'), no_ext=True)
         runs = [ r for r in os.listdir(root_dir) if re.match(filesystem.RUN_RE,r) ]
         with filesystem.chdir(root_dir):
             for run in runs:
-                fc_date = run.split('_')[0]
-                if misc.days_old(fc_date) > days:
-                    if run in archived_in_swestore:
-                        list_to_delete.append(run)
+                ## Collect all project path in the run folder
+                all_proj_path = glob("{}/Unaligned_*/Project_*".format(run))
+                all_proj_dict = {os.path.basename(pp).replace('Project_','').replace('__', '.'): pp for pp in all_proj_path}
+                closed_projects = get_closed_projects(all_proj_dict.keys(), pcon, days)
+                ## Only proceed cleaning the data for closed projects
+                for closed_proj in closed_projects:
+                    if dry_run:
+                        logger.info('Project {} in closed for {} days, will remove fastq files from run {}'.format(closed_proj, days, run))
                     else:
-                        logger.warn("Run {} is older than {} days but not in "
-                                    "swestore, so SKIPPING".format(run, days))
-
-    ## delete and log
-    for item in list_to_delete:
-        if dry_run:
-            logger.info('Will remove {} from {}'.format(item,root_dir))
-            continue
-        try:
-            shutil.rmtree(os.path.join(root_dir,item))
-            logger.info('Removed project {} from {}'.format(item,root_dir))
-            with open(log_file,'a') as to_log:
-                to_log.write("{}\t{}\n".format(item,datetime.strftime(datetime.now(),'%Y-%m-%d %H:%M')))
-        except OSError:
-            logger.warn("Could not remove path {} from {}"
-                        .format(item,root_dir))
-            continue
+                        for fastq_file in glob("{}/*/*.fastq.gz".format(all_proj_dict[closed_proj])):
```

Comment:
not entirely sure what is going on....
`all_proj_dict[closed_proj]` should contain something like ´F.Vezzi_12_01`, and right now I would like to work on a the folder `/proj/a2010002/archive/RUN/Unaligned_16/Project_F_Vezzi__12_01` 
If this is the case than `"{}/*/*.fastq.gz".format(all_proj_dict[closed_proj])` becomes something like `"F.Vezzi_12_01/*/*.fastq.gz"`